helloworld-shell/script.sh

```diff
@@ -14,8 +15,8 @@
 
 # [START run_helloworld_shell]
 
-#!/bin/bash
```

Comment:
Oh, I thought license header had to be at the very top of file. Maybe that was a constraint in how the golang-samples repo had implemented the header check.

Reply:
The issue here is the shebang can't be inside the region tags?